app/assets/stylesheets/application.css.scss.erb

```diff
@@ -27,6 +27,10 @@ body { padding-top: 60px; }
   margin-bottom: 100px;
 }
 
+table {
```

Comment:
@cantino ,I tried it, and it can work. It only affect in markdown-table of agent's description when the code inside of .well.description. On the contrary, it affect the whole table.
So should I make a new PR?